conda/environment-test-3.8.yml

```diff
@@ -17,6 +23,7 @@ dependencies:
   - pyarrow>=5
   - scipy
   - typing_extensions
+  - llvm-openmp
```

Comment:
not sure if it's worth documenting where this requirement comes from inline in the file with a comment